pyproject.toml

```diff
@@ -29,7 +29,7 @@ __version__ = "{version}"
 asyncio_mode = "auto"
 
 [project.scripts]
-run_audiotrigger = "lsst.ts.audiotrigger:execute_runner"
+run_serial_temperature_scanner = "lsst.ts.audiotrigger:execute_serial_temperature_scanner"
```

Comment:
Any reason why one is named `run_...` and the other `execute_...`? I'd suggest to name them both `run_...` so it is clear that they belong together.

Reply:
I think I just have that pattern by habit. But you're right about consistency so I will do it.